web/releases/tenzir-node-v5.1/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Tenzir Node v5.1: Moving Fields"
+slug: tenzir-node-v5.1
+authors: [dominiklohmann]
+date: 2025-04-25
+tags: [release, node]
+comments: true
+---
+
+[Tenzir Node v5.1][github-release] enhances TQL with ways to access and move fields, bitwise functions, and a native integration with Google Cloud Logging.
```

Comment:
Maybe it's just me (or I'm being pedantic), but "enhances TQL with ways to access and move fields" is missing a word, like "different ways to access and move" fields. There were already ways to access fields.